webapp/src/app/pages/components/knowledge-graph/custom-recommendation-option/custom-recommendation-option.component.ts

```
@@ -0,0 +1,707 @@
+    }
+
+    let normalizedValues: number[] | null = null;
+    let scaledData: number[] | null = null;
+
+    if (factorWeights) {
+      values = Object.values(factorWeights);
+    }
+
+    switch (methodType) {
+      case 'l1':
+        const l1Sum = values.reduce((acc, val) => acc + Math.abs(val), 0);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)